alcs-frontend/src/app/features/board/dialogs/planning-review/planning-review-dialog.component.ts

```diff
@@ -67,6 +70,13 @@ export class PlanningReviewDialogComponent implements OnInit, OnDestroy {
       }
     });
 
+    this.authService.$currentUser.pipe(takeUntil(this.$destroy)).subscribe((currentUser) => {
```

Comment:
I almost want to move this check to some kind CardDialogBaseComponent

Reply:
Done!